tests/Doctrine/Tests/ORM/Functional/ClassTableInheritanceTest.php

```diff
@@ -361,8 +355,9 @@ public function testQueryForInheritedSingleValuedAssociation()
         $this->_em->flush();
         $this->_em->clear();
 
-        $dql = "SELECT m FROM Doctrine\Tests\Models\Company\CompanyManager m WHERE m.spouse = ?1";
-        $dqlManager = $this->_em->createQuery($dql)->setParameter(1, $person->getId())->getSingleResult();
+        $dqlManager = $this->_em->createQuery('SELECT m FROM ' . CompanyManager::class . ' m WHERE m.spouse = ?1')
```

Comment:
I'll do a dry-run on the rebase... if it has too many conflicts I'll drop some changes